nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -175,6 +202,22 @@ def migrate_site_and_region_data_to_locations(apps, schema_editor):
     if Region.objects.exists():
         region_lt = LocationType.objects.create(name="Region", nestable=True)
         create_region_location_type_locations(region_class=Region, location_class=Location, region_lt=region_lt)
+        for region_loc in Location.objects.filter(location_type=region_lt):
+            region_loc.tags = _NautobotTaggableManager(
+                through=extras_models.TaggedItem, model=Location, instance=region_loc, prefetch_cache_name="tags"
+            )
+            region_loc.save()
```

Comment:
What is this doing?

Reply:
This is Gary's code to make `TagManager` available in migrations, so `serialize_object()` called in `ObjectChange` can function properly.

Reply:
Not sure I understand what's happening at the DB level when calling `save()` here, @gsnider2195 can you help me understand?

Reply:
deleted `save()` per discussion in standup it was not needed.